swift/ql/src/queries/Security/ECB-Encryption/ECBEncryption.ql

```diff
@@ -0,0 +1,81 @@
+/**
+ * @name Encryption using ECB
+ * @description Highlights uses of the encryption mode 'ECB'. This mode should normally not be used because it is vulnerable to replay attacks.
```

Comment:
The [preferred pattern](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description) for query descriptions is of the type: "Syntax X causes behavior Y."` Could we rework this so it fits the pattern?

I am not a developer but something in the lines of "Using the ECB encryption mode makes code vulnerable to replay attacks." could do. What do you think?